src/aditiGJUN20/PalindromeString.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Verify given String is palindrome or not. 
+ * Input: radar
+ * Output : Given string radar is palindrome.
+ * Input: technocredits
+ * Output: Given string technocredits is not palindrome.
+*/
+package aditiGJUN20;
+
+public class PalindromeString {
+	boolean isPalindromString(String input) {
+		System.out.println("Input String: "+input);
+		String rev="";
+		for ( int index = input.length() - 1; index >= 0; index-- )
+        rev = rev + input.charAt(index); 
+		
+		if(input.equalsIgnoreCase(rev))
```

Comment:
Better to use equal method. If interviewer asked to checking it without case sensitivity you should equalsIgnoreCase

Reply:
I'll keep that in mind.